app/css/vendors/ip-camera.css

```diff
@@ -0,0 +1,77 @@
+.camera-view {
+  display: flex;
+  flex-direction: column;
+}
+
+.camera-view__body {
+  flex: 1;
+}
+
+/** HACK until we unify markup for all views, reset footer absolute positioning **/
+.camera-view .footer {
```

Comment:
No you're correct, the header styling needs some love and fixing.
Can you do that for all pages instead?